management/jobs/cleanup/config.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<project>
+  <actions/>
+  <description></description>
```

Comment:
Logging in on Safari on macOS doesn't work quite right, when using `/mock-login`. The act of logging in works fine, but the URL it uses is `http://localhost:1234/mock-login?redirect_uri=http%3A//localhost%3A1234/log-in`, which is to say that, after logging in, the user is shown this screen:

![login](https://user-images.githubusercontent.com/656758/54134044-981f2980-43ed-11e9-8f8a-03206fef90c0.png)

This problem does not exist in Chrome or Firefox on macOS.

Reply:
we just needed an e.preventDefault() on the login for so that it didn't redirect back to the login page